vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/DragAndDropSteps.java

```diff
@@ -68,6 +75,54 @@ public void dragAndDropToTargetAtLocation(SearchAttributes origin, Location loca
                     // Wait for DOM stabilization
                     .pause(Duration.ofSeconds(1))
                     .perform();
+        });
+    }
+
+    /**
+     * Simulates drag of the <b>draggable</b> element and its drop at the <b>target</b> element via JavaScript.
+     * <br>
+     * <i>Example</i>
+     * <br>
+     * <code>When I simulate drag of element located `By.xpath(//div[@class='draggable'])` and drop at element located
+     * `By.xpath(//div[@class='target'])`</code>
+     * <p>
+     *   The reason of having this step is that Selenium WebDriver doesn't support HTML5 drag&amp;drop:
+     * </p>
+     * <ul>
+     *   <li><a href="https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/3604">
+     *       Issue 3604: HTML5 Drag and Drop with Selenium WebDriver
+     *       </a>
+     *   </li>
+     *   <li><a href="https://github.com/SeleniumHQ/selenium/issues/1365">
+     *       Issue 1365: Actions drag and drop method
+     *       </a>
+     *   </li>
+     * </ul>
+     * <p>
+     * As workaround for these issue the step simulates HTML5 drag&amp;drop via JavaScript. There is no difference in
+     * actual drag&amp;drop and its simulation via JavaScript from the functional side.
+     * </p>
+     * @param draggable draggable element
+     * @param target target element
+     */
+    @When("I simulate drag of element located `$draggable` and drop at element located `$target`")
+    public void simulateDragAndDrop(SearchAttributes draggable, SearchAttributes target)
+    {
+        performDragAndDrop(draggable, target, (draggableElement, targetElement) ->
+                // See gist for details: https://gist.github.com/valfirst/7f36c8755676cdf8943a8a8f08eab2e3
```

Comment:
perhaps it makes sense to put this link into script

Reply:
I've thought about it, but this script is going to be executed in browser, so I think it's better to keep it clean